prototypes/anim_polygons_data/README.md

```diff
@@ -0,0 +1,59 @@
+# Anim-polygons data creation pipeline
```

Comment:
While this file is necessary for the user to know how to convert `.blender` into `.svg`, why bother user at all?

I think it'd be better to replace it with a simple `Makefile` and perhaps a simple shell script:

```make
all: scene.svg

docker:
        docker build -t anim-polygons .

%.svg: %.blender
        run a set of commands inside docker to process $< into $@ ins

clean:
        remove all generated files

.PHONY: docker all clean
```